analytics_data_api/v0/tests/views/__init__.py

```diff
@@ -12,3 +15,27 @@ def setUpClass(cls):
         cls.course_id = DEMO_COURSE_ID
         cls.course_key = CourseKey.from_string(cls.course_id)
         super(DemoCourseMixin, cls).setUpClass()
+
+
+class VerifyNoCourseIdMixin(object):
```

Comment:
Alternatively, something like [`test_bad_request`](https://github.com/edx/edx-analytics-data-api/pull/93/files#diff-7bb85fd0aed6837c1d0d4834f7ff1878R412) might be a little more generic/reusable.